ST1/Arrays/task5.js

```diff
@@ -0,0 +1,6 @@
+//Отсортировать массив [1,6,7,8,3,4,5,6] по убыванию
+
+let arr = [1,6,7,8,3,4,5,6]
+arr.sort();
```

Comment:
if you will sort array like [1,6,10,2,3,20] with this method you get [1,10,2,20,3] because now you sorting string, you need to sort numbers.